### PR TITLE
[WIP] Fixing debug builds on mac

### PR DESCRIPTION
**Pull request by jumde**

### CODE REVIEW

File: patches/master_patch.patch [hunk 1]
```diff
@@ -254,6 +254,42 @@ index cb8bd43871271083038b0555340d4c328f9b8bae..1b442fbf4dfebfc709e90021bb1600a9
...
+@@ -280,31 +280,6 @@ void PermissionDecisionAutoBlocker::UpdateFromVariations() {
+                            &g_blacklist_embargo_days, kDefaultEmbargoDays);
+ }
+ 
+-void PermissionDecisionAutoBlocker::CheckSafeBrowsingBlacklist(
+-    content::WebContents* web_contents,
+-    const GURL& request_origin,
+-    ContentSettingsType permission,
+-    base::Callback<void(bool)> callback) {
+-  DCHECK_EQ(CONTENT_SETTING_ASK,
```
Comment (darkdh) [inline patches/master_patch.patch]: And just leave the implementation empty, do not remove the definition

File: atom/browser/atom_browser_main_parts.cc [hunk 2]
```diff
@@ -208,50 +208,6 @@ int AtomBrowserMainParts::PreCreateThreads() {
   // any callbacks, I just want to initialize the mechanism.)
   SecKeychainAddCallback(&KeychainCallback, 0, NULL);
 #endif  // defined(OS_MACOSX)
-  auto feature_list = base::FeatureList::GetInstance();
```
Comment (bridiver) [inline atom/browser/atom_browser_main_parts.cc]: why were these removed? This would cause crashes and break important functionality

**Comment by jumde** [comment]
Not needed
